test/suites/network/offline.robot

```diff
@@ -86,7 +86,7 @@ Pod Should Be Reachable Via Ingress
     ...    Run Guest Process    ${GUEST_NAME}
     ...        bash
     ...        -c
-    ...        curl --fail -I -H \"Host: hello-microshift.cluster.local\" ${NODE_IP}:80/principal
+    ...        curl --fail -I --max-time 15 -H \"Host: hello-microshift.cluster.local\" ${NODE_IP}:80/principal
```

Comment:
We have another `Run Process` [here](https://github.com/openshift/microshift/blob/main/test/resources/microshift-network.resource#L60) that uses the builtin keyword timeout, should we unify?

Reply:
These are not the same functions. The current test using `Run Guest Process` keyword because it must use qemu protocol to connect to a host with a network card.